lib/galaxy/tools/deps/__init__.py

```diff
@@ -175,6 +176,12 @@ def build_cache(self, requirements, **kwds):
         resolved_dependencies = self.requirements_to_dependencies(requirements, **kwds)
         cacheable_dependencies = [dep for req, dep in resolved_dependencies.items() if dep.cacheable]
         hashed_requirements_dir = self.get_hashed_requirements_path(cacheable_dependencies)
+        if kwds.get('force_rebuild', False) and os.path.exists(hashed_requirements_dir):
```

Comment:
@mvdbeek What happens if an admin installs a tool which has the same set of requirements of a previously installed-and-cached environment (and `force_rebuild` is False)?